drivers/hmis/app/models/hmis/form/definition_validator.rb

```diff
@@ -11,7 +11,9 @@ def self.perform(...)
 
   # @param [Hash] document is a form definition document {'item' => [{...}] }
   # @param [role] role of the form as string ('INTAKE', 'CLIENT', etc). If not provided, HUD rule validation will not occur.
-  def perform(document, role = nil)
+  # @param [owner_type] owner_type of the form, such as Hmis::Hud::CustomAssessment
```

Comment:
Owner can be inferred from the role, maybe we can do that instead of passing it in?

One way we do this sometimes is to define a method as both a class method and instance method:
```
// defintion.rb
  def self.owner_class_for_role(role)
    return Hmis::Hud::CustomAssessment if ASSESSMENT_FORM_ROLES.include?(role.to_sym)

    return unless FORM_ROLE_CONFIG[role.to_sym].present?

    FORM_ROLE_CONFIG[role.to_sym][:owner_class].constantize
  end

  def owner_class
    self.class.owner_class_for_role(role)
  end


//definition_validator.rb

owner_type = Hmis::Form::Definition.owner_class_for_role(role).sti_name
```